README.md

```diff
@@ -127,8 +127,13 @@ state_machine :review_state, prefix: 'some_prefix' do
 end
 ```
 
-Please note that the `:prefix` option will only be applied on state constants and state methods.
-Consequently, developers must take special care when naming state events to ensure clarity and avoid conflicts.
+The `prefix` option is specifically designed to apply only to state constants and state methods.
+This functionality allows the gem to automatically prepend the specified prefix to these elements,
+ensuring a consistent and recognizable naming convention across your codebase.
+However, it is important to note that the `prefix` option does not extend to state events.
+The rationale behind this distinction lies in the origin of these elements: while state constants and state methods
+are predefined by the gem, state events are defined by the developer. Consequently, it is the developer's
+responsibility to name state events appropriately to maintain clarity and avoid conflicts.
```

Comment:
I'd suggest boiling it down to the essence, maybe like this?

```suggestion
Note: The `prefix` option is designed to apply only to constants and state methods **defined by the gem**.

**State event names** are manually defined by the developer and thus not altered even if the `prefix` 
option is used. It's advised to review them regarding potential naming collision and clarity when
 introducing the `prefix` method.
```

Reply:
LGTM!